### DISCUSSION TITLE
Stepping down as a maintainer

### DISCUSSION THREAD

**Discussion by fzadikian** [rocm-arch/rocm-arch#976] [General]
Hello, I'd like to announce that I am no longer contributing to python-torchaudio-rocm, since my amd gpu is not compatible anymore and I am currently using an nvidia gpu for compute.

If I could get some help in handing over the project (especially the aur package from [here](https://aur.archlinux.org/pkgbase/python-torchaudio-rocm/)) I would gladly appreciate it.

Thank you!

**Comment by acxz** [comment]
@FranGamer1892 if you could add me (`acxz`) as a co-maintainer on the AUR package and then disown the package, I will be made maintainer of the package on the AUR. After you hand maintainership over to me, I'll remove you from the github repo as well.

Thanks so much for your work on creating and maintaining the package so far!

**Reply by fzadikian** [reply]
@acxz Done, you should be the maintainer now. I'm sorry that the package was left unmaintained for a while now. It would be great if you could update it to the latest release. Thank you for giving me the opportunity to host the package here! :smile:

**Reply by GUUser91** [reply]
@acxz
Can you fix the python-torchaudio-rocm build problem?
https://github.com/rocm-arch/python-torchaudio-rocm/issues/5
https://github.com/rocm-arch/python-torchaudio-rocm/issues/4
